packages/opentelemetry/src/spanExporter.ts

```diff
@@ -144,69 +151,72 @@ function parseSpan(span: ReadableSpan): { op?: string; origin?: SpanOrigin; sour
   return { origin, op, source };
 }
 
-function createTransactionForOtelSpan(span: ReadableSpan): Transaction {
-  // eslint-disable-next-line deprecation/deprecation
-  const hub = getSpanHub(span) || getCurrentHub();
-  const spanContext = span.spanContext();
-  const spanId = spanContext.spanId;
-  const traceId = spanContext.traceId;
-  const parentSpanId = span.parentSpanId;
-
-  const parentSampled = span.attributes[InternalSentrySemanticAttributes.PARENT_SAMPLED] as boolean | undefined;
-
-  const { op, description, data, origin, source } = getSpanData(span);
+function createTransactionForOtelSpan(span: ReadableSpan): TransactionEvent {
+  const { op, description, data, origin = 'manual', source } = getSpanData(span);
   const metadata = getSpanMetadata(span);
   const capturedSpanScopes = getSpanScopes(span);
 
   const sampleRate = span.attributes[SEMANTIC_ATTRIBUTE_SENTRY_SAMPLE_RATE] as number | undefined;
 
-  const attributes = {
+  const attributes = dropUndefinedKeys({
     [SEMANTIC_ATTRIBUTE_SENTRY_SOURCE]: source,
     [SEMANTIC_ATTRIBUTE_SENTRY_SAMPLE_RATE]: sampleRate,
+    [SEMANTIC_ATTRIBUTE_SENTRY_OP]: op,
+    [SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN]: origin,
     ...data,
     ...removeSentryAttributes(span.attributes),
-  };
+  });
 
-  const dynamicSamplingContext = getDynamicSamplingContextFromSpan(span);
+  const { traceId: trace_id, spanId: span_id } = span.spanContext();
+  const parent_span_id = span.parentSpanId;
 
-  const transaction = startTransaction(hub, {
-    spanId,
-    traceId,
-    parentSpanId,
-    parentSampled,
-    name: description,
+  const status = mapStatus(span);
+
+  const traceContext: TraceContext = dropUndefinedKeys({
+    parent_span_id,
+    span_id,
+    trace_id,
+    data: attributes,
+    origin,
     op,
-    startTimestamp: convertOtelTimeToSeconds(span.startTime),
-    metadata: {
+    status: getStatusMessage(status),
+  });
+
+  const transactionEvent: TransactionEvent = {
+    contexts: {
+      trace: traceContext,
+      otel: {
+        // TODO: remove the attributes here?
+        attributes: removeSentryAttributes(span.attributes),
```

Comment:
I _think_ we can stop sending these in the `otel` context, because they are now basically the same thing as the `trace.data` context, but need to double check if that is actually true 😅